pyo3-derive-backend/src/pyclass.rs

```diff
@@ -387,10 +387,10 @@ fn impl_class(
             const FLAGS: usize = #(#flags)|* | #extended;
 
             #[inline]
-            fn type_object() -> &'static pyo3::ffi::PyTypeObject {
+            fn type_object_raw(py: pyo3::Python) -> &'static pyo3::ffi::PyTypeObject {
```

Comment:
Why `raw`?
Since it returns an FFI type?

Reply:
Yeah, having `TypeObject::type_object` and `PyTypeInfo::type_object` have the same name is a little annoying, which is why I wanted to rename this one. I'm happy to revert this or change the name to another new one as you prefer?

Reply:
:+1: